Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -834,4 +834,29 @@ class Testball < Formula
     desc_cache.unlink
     formula_file.unlink
   end
+
+  def test_style
+    foo_file = CoreTap.new.formula_dir/"foo.rb"
+    foo_file.write <<-EOS.undent
+      class Foo < Formula
+        url 'https://example.com/foo-1.0'
+      end
+    EOS
+
+    rubocop_yml = HOMEBREW_LIBRARY/".rubocop.yml"
+    rubocop_yml.write <<-EOS.undent
+      Style/Documentation:
+        Enabled: false
+
+      Style/StringLiterals:
+        EnforcedStyle: double_quotes
+    EOS
+
+    assert_match "Prefer double-quoted strings", cmd_fail("style")
+    assert_match "1 file inspected, 1 offense detected", cmd_fail("style", "#{foo_file}")
```

Comment:
Just curious, why wrap only these 2 lines?
```
assert_match "1 file inspected, 1 offense detected", cmd_fail("style", "#{foo_file}")
assert_match "1 file inspected, 1 offense detected", cmd_fail("style", "foo")
```
In any case, I removed the `Gemfile*` changes and tried wrapping those 2 lines (and all 3 lines). Received the same error in both cases. :-/